frontend/src/components/Header/Header.tsx

```diff
@@ -50,21 +62,64 @@ function Header({ organisation }: HeaderProps) {
     return (
         <AppBar position="static">
             <Toolbar sx={{ justifyContent: "space-between" }}>
-                <Link to="/">
-                    <Box
-                        height={{ xs: "2rem", md: "2.5rem" }}
-                        component="img"
-                        src={THEME_TO_IMAGE[organisation]}
-                        alt="Organisation logo"
-                    />
-                </Link>
-                {showDateAndTime && (
-                    <Typography variant="h4" component="p">
-                        {date}
-                    </Typography>
-                )}
+                <Tooltip followCursor={true} title="Return to home screen">
+                    <Link to="/">
+                        <Box sx={{ alignItems: "center", display: "flex" }}>
+                            <Box
+                                height={{ xs: "2rem", md: "2.5rem" }}
+                                component="img"
+                                src={THEME_TO_IMAGE[organisation]}
+                                alt="Organisation logo"
+                            />
```

Comment:
Would it be possible to add a hover effect to the Organisation Logo too? Or rather, the container of both the home logo and the org logo.

As it's now, it isn't that obvious to a user that the organisation logo is clickable.

Might be a personal preference altough, but alas.